trello_items.py

```diff
@@ -43,21 +60,39 @@ def get_list_name(card_id):
     list = requests.request("GET", url, params=query)
     return list.json()['name']
 
+def get_all_lists_on_board(board_id):
+    """
+    Fetches all lists from a Trello board.
+    Returns:
+        list: The lists on the boards
+    """
+    key = os.getenv('apiKey')
+    token = os.getenv('apiToken')
```

Comment:
Inconsistent with the rest of the file where you do:
```
    key = app.config['KEY']
    token = app.config['TOKEN']
```

Reply:
This is because the end to end calls this method even before the app is created. So, at this point there isn't any app loaded.